monitoring/voltage-monitoring/src/main/java/com/farao_community/farao/monitoring/voltage_monitoring/VoltageMonitoringResult.java

```diff
@@ -41,33 +45,66 @@ static Status fromConstraints(boolean highVoltageConstraints, boolean lowVoltage
     private final Status status;
     private final Map<VoltageCnec, ExtremeVoltageValues> extremeVoltageValues;
     private final Set<VoltageCnec> constrainedElements;
+    private final Map<State, Set<NetworkAction>> appliedCras;
```

Comment:
I do not understand why you filtered Preventive Remedial Actions in the VoltageMonitoring.
The US says that PRAs can be apply, example quote : `Voltage monitoring can affect the preventive state`

Therefore, you should mofidy the VoltageMonitoring code and the name of this Map.